src/components/LocationPin.js

```diff
@@ -14,7 +14,11 @@ export default function LocationPin({
 }) {
   return img ? (
     <>
-      <button className="pin" tabIndex="0">
+      <button
+        className="pin"
+        tabIndex="0"
+        aria-label="location-pin-icon-with-image"
```

Comment:
You can and probably should replace the dashes in this label with spaces. This is going to be read by screen reader and I’m not sure how they handle words with dashes in them 🤔